packages/hash/frontend/src/shared/layout/layout-with-sidebar.tsx

```diff
@@ -49,47 +48,53 @@ export const LayoutWithSidebar: FunctionComponent<LayoutWithSidebarProps> = ({
 
         <Box
           sx={(theme) => ({
-            width: 1,
-            display: "flex",
+            width: "100%",
             position: "relative",
-            flexDirection: "column",
             marginLeft: `-${SIDEBAR_WIDTH}px`,
+            transition: theme.transitions.create("margin", {
+              easing: theme.transitions.easing.easeOut,
+              duration: theme.transitions.duration.enteringScreen,
+            }),
             ...(sidebarOpen && {
-              transition: theme.transitions.create("margin", {
-                easing: theme.transitions.easing.easeOut,
-                duration: theme.transitions.duration.enteringScreen,
-              }),
               marginLeft: 0,
             }),
           })}
         >
-          {banner}
+          <Fade timeout={800} in={!sidebarOpen}>
+            <Tooltip title="Expand Sidebar">
+              <IconButton
+                size="medium"
+                sx={{
+                  position: "absolute",
+                  top: 8,
+                  left: 8,
+                  transform: "rotate(180deg)",
 
-          <Box>
-            <Fade timeout={800} in={!sidebarOpen}>
-              <Tooltip title="Expand Sidebar">
-                <IconButton
-                  size="medium"
-                  sx={{
-                    position: "absolute",
-                    top: 8,
-                    left: 8,
-                    transform: "rotate(180deg)",
+                  "&:hover": {
+                    backgroundColor: ({ palette }) => palette.gray[20],
+                    color: ({ palette }) => palette.gray[60],
+                  },
+                }}
+                onClick={openSidebar}
+              >
+                <SidebarToggleIcon />
+              </IconButton>
+            </Tooltip>
+          </Fade>
 
-                    "&:hover": {
-                      backgroundColor: ({ palette }) => palette.gray[20],
-                      color: ({ palette }) => palette.gray[60],
-                    },
-                  }}
-                  onClick={openSidebar}
-                >
-                  <SidebarToggleIcon />
-                </IconButton>
-              </Tooltip>
-            </Fade>
-
-            <Main>{children}</Main>
-          </Box>
+          <Main
+            sx={({ spacing }) => ({
+              ...(!fullWidth && {
+                padding: spacing(7, 10),
+              }),
+              ...(fullWidth &&
+                !sidebarOpen && {
+                  marginLeft: spacing(5),
+                }),
+            })}
```

Comment:
Something like this is probably much more readable I think? 

```suggestion
            sx={({ spacing }) => ({
              ...(fullWidth ? (
              	!sidebarOpen && { marginLeft: spacing(5) }
              ) : { padding: spacing(7, 10) })
            })}
```